shallow_backup/git_wrapper.py

```diff
@@ -92,6 +93,11 @@ def move_git_repo(source_path, new_path):
 	"""
 	Moves git folder and .gitignore to the new backup directory.
 	"""
+	if os.path.exists(os.path.join(new_path, '.git')) or os.path.exists(os.path.join(new_path, '.gitignore')):
```

Comment:
Let's extract these paths to variables in order to simplify this `if` statement

Reply:
Just noticed that they're declared a few lines down. Move those declarations up and refactor this.